### PR TITLE
#149669999 Create navigation bar

### PR DESCRIPTION
**Pull request by andela-cofor**
#### What does this PR do?
Ensures user can view navigation bar

#### Description of Task to be completed?
* As an andela I should be able to view a navbar when I login

#### How should this be manually tested?
After cloning the repo seed into it.
* Run `yarn install`
* Run `npm start`
* Click on `LOGIN TO CONTRIBUTE` button

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#149669999](https://www.pivotaltracker.com/story/show/149669999)

#### Screenshots (if appropriate)
screen shot 2017-08-01 at 12 28 46 pm

#### Questions:
N/A

### CODE REVIEW

File: src/common/navbar/NavBar.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      }
+    }
+
+    .alert {
+      width: 1.5rem;
+      height: 1.5rem;
+      border: 1px solid map-get($grey, 400);
+      border-radius: 50%;
+    }
+
```
Comment (andela-kakpobome) [inline src/common/navbar/NavBar.scss]: ### Nitpick

Remove this space, it serves no purpose.

File: src/config/index.ts [hunk 2]
```diff
@@ -4,7 +4,11 @@ const allConfig =  {
     API_BASE_URL: '',
     BASE_URL: 'http://learningmap-dev.andela.com:3000',
   },
-  production: {},
+  production: {
+    ANDELA_API_BASE_URL: 'https://api.andela.com',
+    API_BASE_URL: 'https://private-3b35e-learningmapapi.apiary-mock.com',
+    BASE_URL: 'http://learningmap-prod.andela.com:3000',
```
Comment (andela-kakpobome) [inline src/config/index.ts]: ### Critical

This should be the URL of our Heroku production build

**Comment by andela-cofor** [comment]
@andela-kakpobome done

File: src/pages/contributions/ContributionPage.component.tsx [hunk 3]
```diff
@@ -54,7 +54,10 @@ interfaces.IContributionsPageState> {
   public render() {
     const contributions = this.state.contributions;
     return (
-      <div><ContributionList contributions={contributions}/></div>
+
```
Comment (andela-cofor) [inline src/pages/contributions/ContributionPage.component.tsx]: @Andela-Didacus done...

**Pull request merged**